keras/backend/mxnet_backend.py

```diff
@@ -4137,6 +4158,16 @@ def _preprocess_convnd_kernel(kernel, data_format):
     return kernel
 
 
+@keras_mxnet_symbol
+def _preprocess_dw_convnd_kernel(kernel, data_format):
```

Comment:
I think we can use this function - _preprocess_convnd_input() ?

Reply:
preprocess_convnd_input use transpose: [0, 3, 1, 2]
we need (2, 3, 0, 1) due to special kernel construction for separable and depth-wise in keras convolutional.py